apps/music/test/marionette/loading_test.js

```diff
@@ -0,0 +1,76 @@
+/* global require, marionette, setup, suite, test, __dirname */
+'use strict';
+
+var assert = require('assert');
+var Music = require('./lib/music.js');
+var FakeRingtones = require('./lib/fakeringtones.js');
+var FakeControls = require('./lib/fakecontrols.js');
+
+marionette('Music loading', function() {
```

Comment:
This is a very generic name for this file and for this test. Are you planning to add other suites to this file that are not about distinguishing movies from songs?  If not, I'd change the names to be more specific.